sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Features Added
 
 - **Visual Studio Code Credential** has been re-enabled and now supports **broker authentication** using the Azure account signed in via Visual Studio Code. [#45715](https://github.com/Azure/azure-sdk-for-java/pull/45715)
+- `DefaultAzureCredential` allows selecting one of its credentials by name via environment variable `AZURE_TOKEN_CREDENTIALS`. It will use only the selected type at runtime. For example, set `AZURE_TOKEN_CREDENTIALS=WorkloadIdentityCredential` to have `DefaultAzureCredential` use only `WorkloadIdentityCredential`.
```

Comment:
```suggestion
`DefaultAzureCredential` can be configured to use a specific credential type by setting the `AZURE_TOKEN_CREDENTIALS` environment variable. When set, it will only attempt authentication using the specified credential type. For example, setting `AZURE_TOKEN_CREDENTIALS=WorkloadIdentityCredential` will restrict authentication to workload identity only.
```